CHANGES.rst

```diff
@@ -17,6 +17,10 @@ Bug fixes:
 - *add item here*
 
 
+
```

Comment:
minor: should be only one blank line between entries

Reply:
I have removed the superfluous blank line after merging.